web-app/js/source/topicDialog.js

```diff
@@ -640,10 +735,19 @@ function removeDuplicateResource (arr) {
 	return cleaned;
 }
 
-function outerSaveResource () {
+function outerSaveResource(contentData) {
 	'use strict';
 	var imodID = $('#imodID').val();
-	var resource = $('#resourceDataStore').val();
+	// var resource = $('#resourceDataStore').val();
+	var index;
+	var index1;
```

Comment:
`index1` will throw a lint error
`outerIndex` and `innerIndex` will pass
or `contentIndex` and `resourceIndex` would also pass